src/__tests__/index.js

```diff
@@ -192,3 +204,46 @@ test('forwards props when the GlamorousComponent.rootEl is known', () => {
   )
   expect(ui).toMatchSnapshotWithGlamor()
 })
+
+test('renders a component with theme properties', () => {
+  const Comp = glamorous.div(
+    {
+      color: 'red',
+    },
+    (props, theme) => ({padding: theme.padding}),
+  )
+  expect(
+    render(<Comp theme={{padding: '10px'}} />),
+  ).toMatchSnapshotWithGlamor()
+})
+
+test('passes an updated theme when theme prop changes', () => {
+  const Comp = glamorous.div(
+    {
+      color: 'red',
+    },
+    (props, theme) => ({padding: theme.padding}),
+  )
+  const wrapper = mount(<Comp theme={{padding: '10px'}} />)
+  expect(toJson(wrapper)).toMatchSnapshotWithGlamor(`enzyme.mount`)
```

Comment:
Could we give both of these a more informative name rather than simply `enzyme.mount`. Now that we have several tests in this file it may be useful to have it be: `'with padding 10px theme'` and then `'with theme prop of padding 20px'` or something. That way reviewing the snapshots is easier.